lib/fourchette/heroku.rb

```diff
@@ -30,6 +30,12 @@ def client
     @heroku_client ||= PlatformAPI.connect(api_key)
   end
 
+  def legacy_client
+    api_key = ENV['FOURCHETTE_HEROKU_API_KEY']
+    ENV['HEROKU_API_KEY'] = api_key # necessary for Heroku::Auth.password to work
```

Comment:
I couldn't find support in their platform API for the migrated PG backups, so I was forced to use the ruby client. One of the problems with it is that its Heroku::Client::HerokuPostgresql implicitly uses Heroku::Auth, which tries to read credentials from stdin. Only means I could see of bypassing this behavior is setting this environment variable before it does so.